cmd/convox/Makefile

```diff
@@ -6,13 +6,16 @@ work   := $(gopath)/src/github.com/convox/convox
 all: build
 
 build:
-	mkdir pkg/
+	mkdir -p pkg/
 	export CGO_ENABLED=0
-	env GOOS=linux GOARCH=amd64 go build -ldflags "-X main.version=$(VERSION) -X main.image=$(IMAGE)" -o pkg/convox-linux-amd64
-	env GOOS=linux GOARCH=arm64 go build -ldflags "-X main.version=$(VERSION) -X main.image=$(IMAGE)" -o pkg/convox-linux-arm64
-	env GOOS=darwin GOARCH=amd64 go build -ldflags "-X main.version=$(VERSION) -X main.image=$(IMAGE)" -o pkg/convox-darwin-amd64
-	env GOOS=darwin GOARCH=arm64 go build -ldflags "-X main.version=$(VERSION) -X main.image=$(IMAGE)" -o pkg/convox-darwin-arm64
+	env GOOS=linux GOARCH=amd64 go build -tags=hidraw -ldflags "-X main.version=$(VERSION) -X main.image=$(IMAGE)" -o pkg/convox-linux-amd64
+	env GOOS=linux GOARCH=arm64 go build -tags=hidraw -ldflags "-X main.version=$(VERSION) -X main.image=$(IMAGE)" -o pkg/convox-linux-arm64
+	env GOOS=darwin GOARCH=arm64 go build -tags=hidraw -ldflags "-X main.version=$(VERSION) -X main.image=$(IMAGE)" -o pkg/convox-darwin-arm64
 
+	go env -w GO111MODULE=auto
+	tar cz -C ../.. . | docker run -v $(gopath)/src/github.com/convox/convox:/convox -i ubuntu tar xz -C /convox
+	mkdir -p $(work) && cd $(work) && env GOPATH=$(gopath) $(shell go env GOPATH)/bin/xgo -go 1.16.x -pkg cmd/convox -out cmd/convox/pkg/convox -targets 'darwin/amd64' -ldflags "-X main.version=$(VERSION) -X main.image=$(IMAGE)" .
+	mkdir -p pkg && docker run -v $(gopath):/gopath -i ubuntu tar czv /gopath/src/github.com/convox/convox/cmd/convox/pkg | tar xzv -C pkg --strip-components 8
```

Comment:
Building with the native go build was not working on macOS, I had to change back to use `xgo`. 
We are using the native go build on Linux version because does not work with `xgo`.